_stbt/core.py

```diff
@@ -2619,17 +2616,16 @@ def test_wait_for_motion_half_motion_int():
 @contextmanager
 def _fake_frames_at_half_motion():
     class FakeDisplay(object):
-        def gst_samples(self, _timeout_secs=10):
+        def frames(self, _timeout_secs=10):
+            from _stbt.gst_utils import CapturedFrame
             data = [
                 numpy.zeros((2, 2, 3), dtype=numpy.uint8),
                 numpy.ones((2, 2, 3), dtype=numpy.uint8) * 255,
             ]
             for i in range(10):
-                buf = Gst.Buffer.new_wrapped(data[(i // 2) % 2].flatten())
-                buf.pts = i * 1000000000
-                yield gst_sample_make_writable(
-                    Gst.Sample.new(buf, Gst.Caps.from_string(
-                        'video/x-raw,format=BGR,width=2,height=2'), None, None))
+                yield CapturedFrame(
+                    data[(i // 2) % 2].flatten().reshape((2, 2, 3)),
```

Comment:
Why is this flatten + reshape necessary, if `data[0]` and `data[1]` are already (2,2,3) numpy arrays?